packages/ckeditor5-image/src/image/replaceimagesourcecommand.ts

```diff
@@ -34,6 +43,15 @@ export default class ReplaceImageSourceCommand extends Command {
 		this.value = this.isEnabled ? element.getAttribute( 'src' ) as string : null;
 	}
 
+	/**
+	 * Register callback which will be executed after command execution.
+	 *
+	 * @param callback Callback which will be called after command execution.
+	 */
+	public registerImageCallback( callback: ( writer: Writer, image: Element ) => void ): void {
```

Comment:
It sounds too generic. The method name should include the purpose of those callbacks.